pandas/core/frame.py

```diff
@@ -825,9 +825,9 @@ def items(self):
             for i, k in enumerate(self.columns):
                 yield k, self._ixs(i, axis=1)
 
-    @Appender(_shared_docs["items"] % "Returns\n        -------")
+    @Appender(_shared_docs["items"] % "Yields\n        -------")
```

Comment:
@sameshl you can actually remove this `% Yields` now, and directly edit this in the docstring above. This was only done so that `items` and `iteritems` could have a different title in the docstring (returns vs yields)

Reply:
Ahh. Nice suggestion @jorisvandenbossche. Will make the required change.